src/mutable-source-code.ts

```diff
@@ -1,7 +1,7 @@
-import ts = require('typescript');
+import * as ts from 'typescript';
```

Comment:
Alternatively, exclude the third commit from the merge. I only added it as a minor quality of life fix, where as the other two are where the functionality I need added lies.